### PR TITLE
enable TLS connection from pgbouncer to hosted PG

### PR DESCRIPTION
**Pull request by fuziontech**
## Description
This is required if you use hosted PostgreSQL like RDS

### CODE REVIEW

File: charts/posthog/values.yaml [hunk 1]
```diff
@@ -344,6 +344,8 @@ ingress:
 postgresql:
   # -- Install postgres server on kubernetes (see below)
   enabled: true
+  # -- sslmode for connection - defaults to prefer
```
Comment (macobo) [inline charts/posthog/values.yaml:347]: Worth adding to the comment, what are the valid values?

**Comment by sjmadsen** [comment]
This PR has been open for over two months and is a blocker for externally hosted Postgres. Is there a plan for when it will be merged?

File: charts/posthog/templates/_helpers.tpl [hunk 2]
```diff
@@ -133,7 +133,7 @@ Set postgres password b64
 Set postgres URL
 */}}
 {{- define "posthog.postgresql.url" -}}
-    postgres://{{- .Values.postgresql.postgresqlUsername -}}:{{- template "posthog.postgresql.password" . -}}@{{- template "posthog.postgresql.host" .  -}}:{{- template "posthog.postgresql.port" . -}}/{{- .Values.postgresql.postgresqlDatabase }}
+    postgres://{{- .Values.postgresql.postgresqlUsername -}}:{{- template "posthog.postgresql.password" . -}}@{{- template "posthog.postgresql.host" .  -}}:{{- template "posthog.postgresql.port" . -}}/{{- .Values.postgresql.postgresqlDatabase -}}
```
Comment (tiina303) [inline charts/posthog/templates/_helpers.tpl:136]: don't we need the new line in the end here?

File: charts/posthog/values.yaml (hunk 1 above)
Comment (tiina303) [reply charts/posthog/values.yaml:347]: pretty easy to find on Google and that could change in theory if anything link to https://www.pgbouncer.org/config.html ?

**Review by tiina303 — APPROVED** [review]
fix templates file before merging

**Comment by guidoiaquinti** [comment]
> This PR has been open for over two months and is a blocker for externally hosted Postgres. Is there a plan for when it will be merged?

@sjmadsen you can already use an externally hosted PostgreSQL using TLS by simply passing custom env variables to the pgbouncer deployment via `pgbouncer.env`. Example:

```yaml
pgbouncer:
  env:
  - name: SERVER_TLS_SSLMODE
    value: "your_value"
```

This PR only makes explicit the TLS configuration. @hazzadous I honestly don't see the value of adding this explicit variable but maybe I am missing something. Are you 👍 if I close without merging?

**Comment by hazzadous** [comment]
> Are you 👍 if I close without merging?

I'm ok with that. We can make sure setting the env is documented.

**Pull request closed without merging**